src/hydra/dataProvider.test.js

```diff
@@ -218,4 +218,117 @@ describe('Transform a React Admin request to an Hydra request', () => {
         ]);
       });
   });
+
+  test('React Admin create multipart override', async () => {
+    await dataProvider.introspect();
+
+    return dataProvider
+      .create('resource', {
+        data: {
+          bar: 'baz',
+          foo: 'foo',
+          formEnctype: 'multipart',
+        },
+      })
+      .then(() => {
```

Comment:
Why not using async/await syntax?

Reply:
Sure. Updated tests to use the async/await syntax (also ajusted previous tests on dataProvider.test.js to comply).

Tks for the feedback.

Reply:
Done.